### PR TITLE
Add a simple Dockerfile with all dependencies

### PR DESCRIPTION
**Pull request by PiotrZierhoffer**
I'm not really sure if this should be merged. I created this Dockerfile and I use it for my development. Maybe someone will find it useful.

But I'm ok with rejecting this PR if you don't see it fitting in this repo.

Build the dockerfile with `cd scripts; docker build -t cfu-playground .`

Run with `docker run -ti cfu-playground`

### CODE REVIEW

File: scripts/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM debian:testing
+
+ENV RISCV_DIR=/toolchain/riscv64-unknown-elf-gcc-8.3.0-2020.04.1-x86_64-linux-ubuntu14/bin
```
Comment (irvifa) [inline scripts/Dockerfile:3]: Won't this only needed during the build time? In that time probably we can change it to `ARG`
```suggestion
ARG RISCV_DIR=/toolchain/riscv64-unknown-elf-gcc-8.3.0-2020.04.1-x86_64-linux-ubuntu14/bin
```

File: scripts/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ENV PATH="/renode:${RISCV_DIR}:${PATH}"
+ENV WORKDIR=/CFU-Playground
```
Comment (irvifa) [inline scripts/Dockerfile:5]: Won't this only needed during the build time? In that time probably we can change it to `ARG`
```suggestion
ARG WORKDIR=/CFU-Playground
```

File: scripts/Dockerfile (hunk 1 above)
Comment (PiotrZierhoffer) [reply scripts/Dockerfile:3]: This variable is listed by the README, but I'm not fully sure it's needed. I will double check it.

Comment (tcal-x) [reply scripts/Dockerfile:3]: Correct, RISCV_DIR is not needed or used.

**Pull request merged**